CONTRIBUTING.md

```diff
@@ -9,3 +9,13 @@ $ gofmt -w ./
 $ staticcheck ./...
 $ go test -race ./...
 ```
+
+## Pre-commit hook
+
+Whenever you commit some changes to this repository, our pre-commit hook will automatically
+run on your computer.
+
+To enable this hook to run on your computer, just run:
+```sh
+git config core.hooksPath .githooks
```

Comment:
Just to keep in sync with other examples in this same file and in the repo:

```suggestion
```console
$ git config core.hooksPath .githooks
```